app/assets/stylesheets/components/_background.scss

```diff
@@ -0,0 +1 @@
+.bg-gray-lighter { background-color: $gray-lighter; }
```

Comment:
We have `bg-light-blue` and `sm-bg-light-blue` in `util.scss` so we might want to consolidate our custom background colors.